assets/src/stories-editor/components/higher-order/with-right-click-handler.js

```diff
@@ -57,8 +57,22 @@ const applyWithSelect = withSelect( ( select, props ) => {
 		}
 		const relativePositionX = event.clientX - wrapperDimensions.left;
 		const relativePositionY = event.clientY - wrapperDimensions.top - toolBarHeight;
+		const clientID = getCurrentPage();
```

Comment:
Minor: we usually use `clientId`, not `clientID` for naming. But since this is only used once there's little need for an extra variable anyway.

Reply:
Fixed in e308d19